packages/docusaurus-plugin-content-docs/src/translations.ts

```diff
@@ -111,7 +113,22 @@ function getSidebarTranslationFileContent(
     ]),
   );
 
-  return mergeTranslations([categoryContent, linksContent]);
+  const docs = collectSidebarDocItems(sidebar)
+    .concat(collectSidebarRefs(sidebar))
+    .filter((item) => item.translatable);
+  const docLinksContent: TranslationFileContent = Object.fromEntries(
+    docs.map((doc) => [
+      `sidebar.${sidebarName}.docLink.${doc.label!}`,
```

Comment:
why is the key "docLink" instead of just "doc"? I mean, we use "category" and not "categoryLink". It looks better to use the exact same term as the sidebar item type?

Was wondering if we don't want to use different keys for "doc" and "ref"?

Reply:
> Was wondering if we don't want to use different keys for "doc" and "ref"?

Not necessarily. A "ref" item is always accompanied by a "doc" item and we probably want to reuse the label for the two items, if they are the same.

Reply:
In that case we would need to differentiate each label independently. ref/doc is a pretty arbitrary cutoff, but each label being translated independently is... not too weird, just inconvenient.